Error: command failed (missing --plugin)

While this doesn't seem to be reflected in VSCode's UI, after accidentally checking *developer console*, I noticed that there's wakatime seldom fails to run (copy from console log):
```
[WakaTime] [ERROR] Error: Command failed: python3 /home/psukys/.vscode/extensions/wakatime.vscode-wakatime-1.2.3/out/wakatime-master/wakatime/cli.py --file /data/Documents/Projects/TUM/Thesis/code/ops-ds2os/opsds2os/nlsm.py --plugin vscode/1.29.1 vscode-wakatime/1.2.3 --alternate-project ops-ds2os
```
Trying to run the same command in console, error is more verbose:
```
wakatime: error: unrecognized arguments: vscode-wakatime/1.2.3
```

I believe the problem is missing `--plugin` before `vscode-wakatime/1.2.3` as there's no more aforementioned error by using it.

--- a/src/extension.ts
+++ b/src/extension.ts
@@ -272,9 +272,9 @@ export class WakaTime {
         this.dependencies.getPythonLocation(pythonBinary => {
           if (pythonBinary) {
             let core = this.dependencies.getCoreLocation();
-            let user_agent =
+            let user_agent = 
               this.agentName + '/' + vscode.version + ' vscode-wakatime/' + this.extension.version;
-            let args = [core, '--file', file, '--plugin', user_agent];
+            let args = [core, '--file', file, '--plugin', '"' + user_agent + '"'];
             let project = this.getProjectName(file);
             if (project) args.push('--alternate-project', project);
             if (isWrite) args.push('--write');

